docs/glossary.md

```diff
@@ -152,16 +152,16 @@ test or deploy) when that truth changes.
 ## kustomization
 
 The term _kustomization_ refers to a
-`kustomization.yaml` file, or more generally to a
+`kustomization.yaml` / `kustomization.json` file, or more generally to a
```

Comment:
Let's not add `kustomization.json` to this doc, since we didn't do that for `kustomization.yml` or `kustomization`.